srtcore/core.cpp

```diff
@@ -11825,12 +11825,12 @@ void CUDTGroup::syncWithSocket(const CUDT& core)
 {
     // [[using locked(m_GroupLock)]];
 
-    currentSchedSequence(core.ISN());
+    set_currentSchedSequence(core.ISN());
     setInitialRxSequence(core.m_iPeerISN);
 
     // Get the latency (possibly fixed against the opposite side)
     // from the first socket.
-    latency(core.m_iTsbPdDelay_ms*int64_t(1000));
+    set_latency(core.m_iTsbPdDelay_ms*int64_t(1000));
```

Comment:
Why does the comment say "Get the latency", while it is followed by setting the latency?

Reply:
Because "getting" refers to `core.m_iTsbPdDelay_ms` part.